hr_employee_health/views/hr_employee.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!--
+      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+    -->
+
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form.inherit.health</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page name="health" string="Health" groups="hr.group_hr_user">
```

Comment:
Not sure if `hr.group_hr_user` is needed here or not since it is already there on all fields

Reply:
Otherwise v12 would have errored

Reply:
> Not sure if `hr.group_hr_user` is needed here or not since it is already there on all fields

Have a look at https://github.com/odoo/odoo/issues/31744 and https://github.com/odoo/odoo/pull/31646#issuecomment-471469838

Reply:
Do we want to show empty page?

Reply:
Ah okay, I got it. :+1: